raiden/tests/integration/test_echo_node.py

```diff
@@ -20,7 +21,12 @@
 @pytest.mark.parametrize('channels_per_node', [CHAIN])
 @pytest.mark.parametrize('reveal_timeout', [18])
 @pytest.mark.parametrize('settle_timeout', [64])
-def test_event_transfer_received_success(token_addresses, raiden_chain, skip_if_not_udp):
+def test_event_transfer_received_success(
+    token_addresses,
+    raiden_chain,
+    network_wait,
+    skip_if_not_udp,
```

Comment:
If the fix is for matrix, it shouldn't be skipped here

Reply:
it's two parallel things - first problem is usual `raiden_chain` setup bug that occurs if stars are aligned correctly, the other is message sync problem that causes the test to fail in some circumstances

Reply:
Right then, just to keep in mind that, to test it with matrix, this parameter must be removed =)